### PR TITLE
Updated readme.md with Table of content (TOC)

### PR DESCRIPTION
**Pull request by ng29**
Updated readme.md with Table of content (TOC)

Please find PR of the below issue.
Issue : #7 

Cheers,

### CODE REVIEW

**Comment by lukeocodes** [comment]
hey @ng29 I dont think this README requires a toc, but it's not spam. so I am closing this PR without labels :) Thank for your contribution

**Comment by ng29** [comment]
> hey @ng29 I dont think this README requires a toc, but it's not spam. so I am closing this PR without labels :) Thank for your contribution

@lukeocodes 
Just for clarification, i started working on this issue after getting confirmation from @yallen011 and you can check this out in the conversation of issue #7 and after approval i made PR for this issue.

So IMO its not spam PR and moreover I was just contributing to open source contribution in month of Hacktoberfest.

And you closed my PR without merging, quietly disappointed I am :(

PLease look into it, do let me know

File: README.md [hunk 1]
```diff
@@ -8,6 +8,12 @@
 
 This Spring Boot Starter has been provided to help with integrating the [Nexmo Java SDK](https://github.com/Nexmo/nexmo-java) into your [Spring Boot](https://spring.io/projects/spring-boot) project.
 
+ * [Installation](#installation)
+ * [Usage](#usage)
+ * [Customizing the `NexmoClient`](#customization)
```
Comment (yallen011) [inline README.md:13]: Remove the backticks around NexmoClient

File: README.md [hunk 2]
```diff
@@ -8,6 +8,12 @@
...
+ * [Customize Nexmo Client Version](#customization_version)
+ 
+<a name="installation"></a>
```
Comment (yallen011) [inline README.md:16]: HTML anchor tag is not needed. See [here](https://raw.githubusercontent.com/Vonage/vonage-java-sdk/master/README.md) for example on how to create TOC.

File: README.md [hunk 3]
```diff
@@ -32,6 +38,7 @@ For Maven:
 </dependency>
 ```
 
+<a name="usage"></a>
```
Comment (yallen011) [inline README.md:41]: HTML anchor tag is not needed. See [here](https://raw.githubusercontent.com/Vonage/vonage-java-sdk/master/README.md) for example on how to create TOC.

File: README.md [hunk 4]
```diff
@@ -74,6 +81,7 @@ nexmo.creds.application-id=application-id
 nexmo.creds.private-key-contents=contents-of-the-key
 ```
 
+<a name="customization"></a>
```
Comment (yallen011) [inline README.md:84]: HTML anchor tag is not needed. See [here](https://raw.githubusercontent.com/Vonage/vonage-java-sdk/master/README.md) for example on how to create TOC.

File: README.md [hunk 5]
```diff
@@ -94,6 +102,7 @@ public NexmoClient.Builder customNexmoBuilder() {
 ```
 > Note that you must include your credentials as shown in this example. This builder completely replaces the automatically configured one.
 
+<a name="customization_version"></a>
```
Comment (yallen011) [inline README.md:105]: HTML anchor tag is not needed. See [here](https://raw.githubusercontent.com/Vonage/vonage-java-sdk/master/README.md) for example on how to create TOC.

**Review by yallen011 — CHANGES_REQUESTED** [review]
Changes requested. Click on the Refresh icon by my name under **Reviewers** on the top right next to the PR title to notify me that the changes have been made and are ready for review.

File: README.md (hunk 1 above)
Comment (ng29) [reply README.md]: Modified

File: README.md (hunk 3 above)
Comment (ng29) [reply README.md]: Modified

File: README.md (hunk 4 above)
Comment (ng29) [reply README.md]: Modified

File: README.md (hunk 5 above)
Comment (ng29) [reply README.md]: Modified

**Comment by ng29** [comment]
> Changes requested. Click on the Refresh icon by my name under **Reviewers** on the top right next to the PR title to notify me that the changes have been made and are ready for review.

Hey @yallen011 all changes have been made and thanks for pointing as it adds to my knowledge bank.
Good to go for review.

**Review by yallen011 — APPROVED** [review]
Good to go. Thank you @ng29 for contributing to the Nexmo Spring Boot Starter project. Also, if you would like to contribute more to the repo, I have just added a new issue to update the spring boot version and change the project from using the nexmo server sdk dependency to using the vonage server sdk dependency if you would like to take it. It is a basic change that shouldn't take much time to complete and I left complete instructions on what to do in the issue [here](https://github.com/Nexmo/nexmo-spring-boot-starter/issues/10). Happy Hacktoberfest 👻  🎃

**Comment by ng29** [comment]
> Good to go. Thank you @ng29 for contributing to the Nexmo Spring Boot Starter project. Also, if you would like to contribute more to the repo, I have just added a new issue to update the spring boot version and change the project from using the nexmo server sdk dependency to using the vonage server sdk dependency if you would like to take it. It is a basic change that shouldn't take much time to complete and I left complete instructions on what to do in the issue [here](https://github.com/Nexmo/nexmo-spring-boot-starter/issues/10). Happy Hacktoberfest 👻 🎃

Sure @yallen011  do assign me and I will look into it.

**Comment by yallen011** [comment]
Awesome, that's great. Sure will

**Comment by ng29** [comment]
> Awesome, that's great. Sure will

Yeah, just came to know about Vonage Hacktoberfest Reward Giveaway too will fill the form tomorrow, Happy Hacktoberfest👻 🎃👻 🎃

Cheers

**Pull request merged**